frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -161,7 +161,107 @@ export default function ProofOfCompletedService_Web({ data }) {
           <small>(Utah Code 78B-7-101 et seq.)</small>.
         </p>
       </Section>
-      <Section name="4. Service by Third Person (continued)"></Section>
+      <Section name="4. Service by Third Person (continued)">
+        <Checkbox
+          dataKey="case.proofOfCompletedServiceDeliveredForm"
+          label="On"
+        />
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredDate"
+          label="date MM-DD-YYYY"
+        />
+        <p>I went to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredAddressZip"
+          label="(address)"
+        />
+        <p>and I delivered the documents listed in paragraph 1 to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredName"
+          label="(name)"
+        />
+        <p>
+          who is <small>(Check one.)</small>
+        </p>
+        <Checkbox
+          dataKey="person.defendant"
+          label="the named defendant/respondent"
+        />
+        <Checkbox
+          dataKey="person.plaintiff"
+          label="the named plaintiff/petitioner"
+        />
+        <Checkbox
+          dataKey="person.proofOfCompletedServicePersonSuitableAge"
+          label="a person of suitable age and discretion residing at that address, which is the named party's residence"
+        />
+        {data.person.proofOfCompletedServicePersonSuitableAge && (
+          <TextArea
+            dataKey="person.proofOfCompletedServicePersonSuitableAgeDescribe"
+            label="Describe why the person lives at the named party's residence and why they are of suitable age and discretion"
+          />
+        )}
+
+        <Checkbox
+          dataKey="person.agent"
+          label="an agent authorized by appointment or by law to receive servie of process on behalf of the named party"
+        />
+        <br></br>
+        <small>
+          If serving a corportion, partnership, or an unincorporated association
+        </small>
+        <Checkbox
+          dataKey="person.officer"
+          label="an officer, a managing agent, general agent"
+        />
+        <br></br>
+        <strong>OR</strong>
+        <Checkbox
+          dataKey="person.authorizedAgent"
+          label="an agent authorized by appointment or by law to receive service of
+          process and by also mailing a copy of the complaint and summons to the
+          named party, if the agent is one authorized by statute to receive process
+          and the statute so requires."
+        />
+        <br></br>
```

Comment:
This is a bit of a nitpick, but I prefer to not use too many `<br />` tags; if you can use styling here I think that would be better. you could use classnames and css like this: https://github.com/JustUtahCoders/utahexpungements.org/blob/master/frontend/app/forms/bank/fillable-form.component.js#L45

Reply:
Agreed on this. CSS is preferred over `<br>` when possible

Reply:
Added css styling.